CONTRIBUTING.md

```diff
@@ -45,6 +45,12 @@ You'll also need [Insta](https://insta.rs/docs/) to update snapshot tests:
 cargo install cargo-insta
 ```
 
+and pre-commit to run some further linter:
```

Comment:
Maybe: "And [`pre-commit` ](https://pre-commit.com/#installation), to run some validation checks:"